### DISCUSSION TITLE
Many lines in the docstring are folded into one line in the tooltip, and whether to fold or not is mysterious

### DISCUSSION THREAD

**Discussion by yvvt0379** [microsoft/pylance-release#2645] [Enhancement]
## Environment data

-   Language Server version: 2021.8.0
-   OS and version: Windows 10 Home 20H2
-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.6

## Expected behaviour

Never fold many lines into one line

## Actual behaviour

Some lines are folded into one line, others not, which makes it harder to read the docstring.

## Logs

```
I don't think it matters here.
```

## Code Snippet / Additional information

```python
I don't think it matters here.
```
Take `super` as example, here's its actual docstring(in the terminal):
![屏幕截图 2021-08-11 220959](https://user-images.githubusercontent.com/62426149/129046349-060efe69-059e-49a6-a2eb-2955b1244dad.png)
However, when I hover on `super`, some lines are folded into one line.
![屏幕截图 2021-08-11 220858](https://user-images.githubusercontent.com/62426149/129046546-c7626046-f53a-404c-9519-628128906327.png)
When a docstring contains patterns like:
```
expression1 -> its meaning
expression2 -> another meaning
...
```
, folding them into one line makes it harder to tell expressions and meanings/explanations apart.

**Comment by yvvt0379** [comment]
And that's just one of the disadvantages of that. As you can see, `@classmethod` and `def cmeth(cls, arg):` were also in one line, which is not a correct syntax in Python.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
